### PR TITLE
docs: docker version and docker platform parameter issue

### LINKED ISSUE #1775: Summary of docker version and docker platform parameter issue
## Documentation Scope
README have not docker version info, that causes some problems in the construction and operation of docker. And mac(m1) users have some problems on docker.

## Describe the Change
update README and give mac(M1) users some suggestions.

## Screenshots
image

### PR DESCRIPTION
**Pull request by abeizn**
fix #1775

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -32,3 +32,6 @@ COPY --from=builder /usr/share/zoneinfo /usr/share/zoneinfo
 ENV PATH="/app/bin:${PATH}"
 
 CMD ["lake"]
+
+# Notes: Docker for Mac sets up qemu emulation, you can likely use the amd64 image by adding the --platform=linux/amd64 flag.
```
Comment (klesh) [inline Dockerfile:36]: I think `M1` is missing?

Comment (abeizn) [reply Dockerfile]: Updated

File: Dockerfile [hunk 2]
```diff
@@ -32,3 +32,6 @@ COPY --from=builder /usr/share/zoneinfo /usr/share/zoneinfo
 ENV PATH="/app/bin:${PATH}"
 
 CMD ["lake"]
+
+# Notes: Docker for Mac(M1) sets up qemu emulation, you can likely use the amd64 image by adding the --platform=linux/amd64 flag.
```
Comment (klesh) [inline Dockerfile:36]: `likely` doesn't feel right. Maybe `you can try to use ....` is a better expression?

Comment (abeizn) [reply Dockerfile]: Okay, updated.

**Review by klesh — APPROVED** [review]
LGTM

**Pull request merged**